output_4.1.txt

```diff
@@ -0,0 +1,33 @@
+TASK: 4.1
```

Comment:
+ I do not see a query that creates the index
+ the `explain` query demonstrates that you actually have no index